kujira/assetlist.json

```diff
@@ -1107,6 +1107,54 @@
           "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/kujira/images/bKUJI.png"
         }
       ]
+    },
+    {
+      "denom_units": [
+        {
+          "denom": "ibc/82B298F9E74A9A211C55C67C7AB0C790836BD43672D5DD830F7AE2C5D010AD7D",
+          "exponent": 0,
+          "aliases": [
+            "rstk"
+          ]
+        },
+        {
+          "denom": "rstk",
+          "exponent": 6
+        }
+      ],
+      "type_asset": "ics20",
+      "base": "ibc/82B298F9E74A9A211C55C67C7AB0C790836BD43672D5DD830F7AE2C5D010AD7D",
+      "name": "Restake DAO Token",
+      "display": "RESTAKE",
```

Comment:
Please fix the "display" value for each asset. They must refer to a denom_unit. It this case, it should be "rstk". This applies to all the assets.